pipelines/setup.yml

```diff
@@ -37,7 +37,7 @@ jobs:
   - name: self-update
     plan:
       - get: paas-release-ci
-        trigger: {{auto_trigger}}
+        trigger: true
```

Comment:
I'm not sure if it's comparable because this pipeline is short and quite meta - you have to run it to see changes to the build and deploy pipelines, unlike paas-cf where those pipelines are uploaded from your own machine.